layouts/partials/encounter/homepage.html

```diff
@@ -0,0 +1,36 @@
+{{ $encounterPages := (.site.GetPage "/encounters").Pages }}
+{{ $savage := where $encounterPages ".Params.encounter_category" "savage"}}
+{{ $ultimate := where $encounterPages ".Params.encounter_category" "ultimate"}}
+<div class="flex flex-col">
+  <div class="w-full mb-24">
+    <div class="flex flex-row items-end">
+      <h1 class="h-shadow flex-grow text-3xl sm:text-5xl uppercase tracking-wider font-head font-bold mb-8">Savage Tiers</h1>
+    </div>
+    <div class="w-full">
+      {{ range $savage.GroupByParam "tier_weight" }}
+        {{ range .Pages.GroupByParam "series_name"}}
```

Comment:
Understanding how `PageGroups` and `Pages` work was a tough one to get this whole logic figured out

Reply:
There is an actual ton of logic around page fetching, parsing, and metadata that, as I read through the docs, makes me think that the hugo people intentionally made it flexible like this so we could start treating pages as fetch-able DB resources